<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<<<<<<< HEAD
<testsuite hostname="Balaji-Note" name="com.esri.test.auto.testcases.TC02_ImageSubPortalPage1" tests="1" failures="0" timestamp="1 Sep 2016 20:28:35 GMT" time="0.000" errors="1">
=======
<testsuite hostname="IMPC3478" name="com.esri.test.auto.testcases.TC02_ImageSubPortalPage1" tests="1" failures="0" timestamp="6 Sep 2016 16:50:16 GMT" time="0.000" errors="1">
>>>>>>> branch 'master' of https://github.com/Esribalajih/ESRI-SEL-Auto
  <testcase name="imageSubmissionPortal" time="0.000" classname="com.esri.test.auto.testcases.TC02_ImageSubPortalPage1">
    <error type="org.testng.TestNGException" message="
Method public void com.esri.test.auto.testcases.TC02_ImageSubPortalPage1.imageSubmissionPortal(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.Throwable requires a @DataProvider named : Fetchdata">
      <![CDATA[org.testng.TestNGException: 
Method public void com.esri.test.auto.testcases.TC02_ImageSubPortalPage1.imageSubmissionPortal(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) throws java.lang.Throwable requires a @DataProvider named : Fetchdata
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:265)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:418)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1240)
	at org.testng.internal.Invoker.createParameters(Invoker.java:980)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1070)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- imageSubmissionPortal -->
</testsuite> <!-- com.esri.test.auto.testcases.TC02_ImageSubPortalPage1 -->
